cli.js

```diff
@@ -8,17 +8,38 @@ const cli = meow(`
 	  $ slugify <string>
 
 	Options
-	  --separator=<string>  Word separator [Default: -]
+	  --separator=<string>                    Word separator [Default: -]
+	  --lowercase=<string>                    Make the slug lowercase. [Default: true]
+	  --decamelize=<string>                   Convert camelCase to separate words [Default: true]
+	  --preserve-leading-underscore=<string>  If your string starts with an underscore, it will be preserved in the slugified string. [Default: false]
```

Comment:
You need to update the readme output too.

Reply:
Also, all of these except `separator` are boolean arguments and should not have a value.

Reply:
How would you like to address this given we need to set them to false (default being true for `lowercase` and `decamelize`?

`--disable-lowercase`, `--disable-decamelize` and `--preserve-leading-underscore`?

Reply:
`--no-lowercase`. `meow` already handles this convention automatically.